### DISCUSSION TITLE
Source Han HC vs TC usage

### DISCUSSION THREAD

**Discussion by mulliganaceous** [adobe-fonts/source-han-sans#462] [General]
### Prerequisites

Traditional Chinese text is often written completely differently from Simplified Chinese text necessitating different tailored fonts to adhere to the desired standards. If no specific language tag is added, you will usually see the Simplified Chinese or the Japanese standard for Han characters.

![dissonantGoogle](https://github.com/adobe-fonts/source-han-sans/assets/29824582/c934d51e-dc8c-4890-a6f8-f66c2fde116c)

### Description

The HC (Traditional Chinese for Hong Kong) standard is probably rarely seen. I have not seen print or handwritten media following the HC standard despite seeing many Traditional Chinese media with the Xinzixing (SC), TC, and Jiuzixing (KR-lite) standard together. I also occasionally saw Simplified Chinese text that incorrectly uses the TC standard in official media such as my ASUS laptop charger.

Additionally, the HC standard, based on _List of Graphemes of Commonly-Used Chinese Characters_, is only a reference, not to be strictly adhered to. Otherwise, HK text may either use SC, TC, or Jiuzixing form.

Only recently will you _might_ see the HC font due to language tags and having the HC font pre-installed on your computer.

**Comment by CoolMarvel43** [comment]
This "issue" that you posted does not even concern Source Han Sans at all. I know you're trying to ask a question to Adobe, but Adobe is very busy preparing for a new release of Source Han Sans and Serif, so I'll give my reply first. Either Adobe doesn't know much about practical Chinese font usage or they are trying to change the situation with regards to language tagging on the Internet so the HK locale of this font can be used more appropriately for Hong Kong use, especially on free desktop operating systems like Linux and mobile devices like Android.

I am also not speaking for Hong Kong as I do not live there, but I think the HK locale was created because the HK government wants a localised standard for education settings. From what I read, it's not mandatory outside of education use, so it's basically up to anyone if they want to use the font or not, and they are responsible if they choose the wrong glyphs (or the wrong font). And speaking of language tagging, it is up to the web developers to implement it properly. Yes, I think there is some ignorance out there because people think Traditional Chinese is only the Taiwan educational standard. But actually it's way more than that, and I don't want to go into details or [I'll be digging](https://github.com/adobe-fonts/source-han-sans/issues/6) [that rabbit hole again](https://github.com/adobe-fonts/source-han-sans/issues/48) and say some things that are not entirely true. Someone more qualified can chime in if it's possible.

As for Microsoft (when you mentioned ASUS, it most likely means a Windows PC), they never bothered to make a HK version of Microsoft Jhenghei (the default UI font for Traditional Chinese since Windows Vista in 2007), because they think it's not important, so it's simply unfortunate that HK users will have to see the Taiwan educational forms and get used to it. For Apple, since 2015, they have included Pingfang HK so that locale will be chosen if the "Traditional Chinese (Hong Kong)" language is used.

Another piece of trivia: Because the Japanese market in the 90s and the early 2000s was bigger and much more important to Microsoft and Apple than all of Greater China, when multilanguage text became possible with Unicode, Japanese came first before Chinese. That's why we sometimes see those semi-traditional glyph shapes of Japanese first. And people just use the default fallback fonts as it is, thinking those Japanese fonts support Traditional Chinese when it's not really the case. Then we can sometimes get an ugly mix of two fonts because the Japanese font doesn't support a particular Traditional Chinese character (e.g. 值 (U+503C), 查 (U+67E5)) or even worse, most of the Simplified Chinese characters. Also, the fact is jiuzixing (舊字形, old-style glyphs) is almost non-existent in computers (save for the Korean fonts which do not cover every single common-use Chinese character if they support them at all), only offline media because of paid fonts or forks of Source Han based off the Korean form.

Consider this issue closed.

**Comment by mulliganaceous** [comment]
I am looking for opinions from HK users regarding seeing HK-specific, TW-specific, or Jiuzixing usage.

**Comment by CoolMarvel43** [comment]
@mulliganaceous I think you can have a look at these links:

### Jiuzixing usage
- https://github.com/adobe-fonts/source-han-sans/issues/6 (half of the posts are in Chinese, you can use Google Translate for that)
- https://github.com/notofonts/noto-cjk/issues/1
- https://github.com/IBM/plex/issues/346

### Hong Kong vs Taiwan educational usage
- https://github.com/adobe-fonts/source-han-sans/issues/18
- https://github.com/adobe-fonts/source-han-sans/issues/32
- https://github.com/adobe-fonts/source-han-sans/issues/48
- https://github.com/notofonts/noto-cjk/issues/71

Most of them are very long, but it's an insight into what the HK users say about their font usage, specifically from users @/hfhchan and @/tamcy AFAIK. I only took a brief look at those links to see if they are appropriate for this issue. I hope this can help.

**Comment by c933103** [comment]
Unless I am missing something, the post at #1 describe the current situation but which part of it is in question?

**Comment by CoolMarvel43** [comment]
> Unless I am missing something, the post at #1 describe the current situation but which part of it is in question?

@c933103 For #1, I think you actually meant notofonts/noto-cjk#1. That #1 in this repository is a completely unrelated issue. You might want to edit your post.

Anyway, notofonts/noto-cjk#1 and #6 are basically about traditional orthography (舊字形) usage versus the modern usage of Taiwan MOE forms (臺灣標準字形). But actually, it’s about the OP over at those two issues begging Adobe and Google to support the former, but unfortunately due to limited glyph space and having to appeal to national standards, they will never be able to support 舊字形. Also, IIRC, 舊字形 is not very well documented and has many variants over the course of history, hence the reason not to support it.

But regardless, the OP of this issue here was originally asking about how the Hong Kong educational glyph usage actually matters on the Internet and offline media, compared to the more widespread usage of old traditional forms and Taiwan MOE forms. Which I think this issue should just be closed.

**Comment by mulliganaceous** [comment]
> > 
> But regardless, the OP of this issue here was originally asking about how the Hong Kong educational glyph usage actually matters on the Internet and offline media, compared to the more widespread usage of old traditional forms and Taiwan MOE forms. Which I think this issue should just be closed.

Yep. This is exactly what I am asking. I am asking if
- Does it matter if you have to use the "HK" version when dealing with Hong Kong media, and "TC" for just Taiwan media?
- Actual photograph examples of offline media that uses the "HK" standard, besides glyph reference books.
    - Educational textbooks (most likely the purpose of the "HK" and "TC" glyphs)
    - Street signs and billboards
    - Other books
